### DISCUSSION TITLE
Libraries

### DISCUSSION THREAD

**Discussion by DemolderSteven** [thieu-b55/Slimme-Digitale-Energie-Meter#4] [Q&A] [answered]
Ik heb een klein beetje hulp nodig met het vinden van de juiste librairies.
kan iemand mij in de juiste richting wijzen?

#include <WiFi.h>
#include <Preferences.h>
#include <AsyncTCP.h>
#include <ESPAsyncWebServer.h>

**Comment by DemolderSteven** [comment, marked as answer]
Oplossing:
Download https://github.com/espressif/arduino-esp32 en kopieer naar de install map van arduino.
Download https://github.com/me-no-dev/ESPAsyncWebServer en hernoem: ESPAsyncWebServer-master ---> ESPAsyncWebServer kopieer deze dan in librairies map.
😁

**Comment by thieu-b55** [comment]
Steven,
ik zie dat je de oplossing al gevonden hebt, of zie ik dat fout?

gr,
thieu

Op za 24 dec. 2022 om 11:16 schreef DemolderSteven ***@***.***

**Reply by DemolderSteven** [reply]
Ik geloof van wel.
was dit nog vergeten bij de oplossing.
download https://github.com/me-no-dev/AsyncTCP en hernoem: AsyncTCP-master ---> AsyncTCP
verplaats deze dan naar librairies map.
correct me if I'm wrong.
Arduino is niet mijn sterkste kant. @thieu-b55

**Comment by thieu-b55** [comment]
Steven,

Steven dat is al een tijdje geleden dat ik die geïnstalleerd heb, vrees dat
ik dat niet meer weet. Maar als het werkt zal het wel ok zijn denk ik .
Interessante website
https://randomnerdtutorials.com/projects-esp32/
met uitleg over ongeveer alles esp32 en esp32 op arduino.

gr,
thieu

Op ma 26 dec. 2022 om 12:43 schreef DemolderSteven ***@***.***

**Reply by DemolderSteven** [reply]
Ok bedankt voor de info
